packages/blade/src/components/Dropdown/DropdownOverlay.web.tsx

```diff
@@ -58,6 +59,99 @@ const AnimatedOverlay = styled(StyledDropdownOverlay)<{
     `,
 );
 
+/**
+ * This function calculates the position of dropdown overlay with respect to dropdown trigger element.
+ * For non-menus (e.g SelectInput), position is flipped if overflow is on bottom.
+ * For menus (e.g. DropdownButton), position is flipped if overflow is on right or bottom.
+ * Additional spacing is added to clientHeight to provide spacing above the dropdown trigger.
+ */
+const getDropdownOverlayPosition = ({
```

Comment:
Oh okay so it wasn't because of type imports. That makes sense now.

Reply:
Yup